packages/babel-plugin-open-source/README.md

```diff
@@ -23,11 +23,25 @@ Add the plugin to your babel config.
 // .babelrc
 {
    "plugins": [
-      "babel-plugin-open-source"
+      "babel-plugin-open-source",
+      "editor:"vscode", // vscode-insiders | sublime | atom
     ]
 }
 ```
 
+Current Supoorted editors are:
```

Comment:
I'd remove this list in favor of the exact string values in the suggestion above ⬆️